batavia/types/Range.js

```diff
@@ -199,6 +199,18 @@ Range.prototype.__getitem__ = function(index) {
     }
 }
 
+
+Range.prototype.__add__ = function(other) {
+  var types = require('../types')
+  if (types.isinstance(other, types.Bool)) {
+    var msg = 'unsupported operand type(s) for +: '
+    throw new exceptions.TypeError.$pyclass(msg +
+                                            '\'' + type_name(this) + '\' and \''
+                                            + type_name(other) + '\'')
```

Comment:
At column 1: [(indent) Expected indentation of 4 spaces but found 2.](http://eslint.org/docs/rules/indent)